Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -48,6 +48,20 @@
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
 
+- type: entity
+  id: AmmoTechFabCircuitboard
+  parent: BaseMachineCircuitboard
+  name: ammo techfab circuit board
+  description: A machine printed circuit board for an ammo techfab
+  components:
+    - type: Sprite
+      state: security
+    - type: MachineBoard
+      prototype: AmmoTechFab
+      requirements:
+        MatterBin: 3
+        Manipulator: 3
```

Comment:
didnt emo say to make these cost 1

Reply:
oh shit, I thought I did this, ill get this fixed quick

Reply:
oh wait, the github isnt updating.. what the heck.

Reply:
its updated now :)